src/renderer/gdi/state.cpp

```diff
@@ -396,7 +396,8 @@ GdiEngine::~GdiEngine()
         // NOTE: not using what GDI gave us because some fonts don't quite roundtrip (e.g. MS Gothic and VL Gothic)
         lf.lfPitchAndFamily = (FIXED_PITCH | FF_MODERN);
 
-        wcscpy_s(lf.lfFaceName, ARRAYSIZE(lf.lfFaceName), FontDesired.GetFaceName());
+        // NOTE: GDI cannot support font faces > 32 characters in length. THIS TRUNCATES THE FONT.

```

Comment:
Do we have any idea if GDI will accept the first 31-32 characters of a font whose name is too long and still resolve the correctish one?